src/components/QuranReader/TafsirView/TafsirModal.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import { useSelector } from 'react-redux';
+
+import TafsirIcon from '../../../../public/icons/tafsir.svg';
+
+import TafsirBody from './TafsirBody';
+
+import EmbeddableContent from 'src/components/dls/EmbeddableContent/EmbeddableContent';
+import PopoverMenu from 'src/components/dls/PopoverMenu/PopoverMenu';
+import { selectSelectedTafsirs } from 'src/redux/slices/QuranReader/tafsirs';
+import { logButtonClick } from 'src/utils/eventLogger';
+import { fakeNavigate, getVerseTafsirNavigationUrl } from 'src/utils/navigation';
+import Verse from 'types/Verse';
+
+type TafsirModalProps = {
+  verse: Verse;
+};
+
+const TafsirModal = ({ verse }: TafsirModalProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { t } = useTranslation();
+  const tafsirs = useSelector(selectSelectedTafsirs);
```

Comment:
We want to change the tafsirs to be a single select, and remove it from the settings menu

Reply:
was chatting with Osama a bit, about the possibility in future whether we will support multiple tafsir or not in the future. 
Maybe it's better to keep array for now to keep the flexibility of possible changes in the future.